.gitignore

```diff
@@ -25,3 +25,6 @@ venv/
 
 # Mac files
 .DS_Store
+
+# Coverage
+.coverage
```

Comment:
```suggestion
.coverage

```

Reply:
Can we add a lefthook to clean these up? 1 and done.

Reply:
perhaps https://dprint.dev/ has something for this

Reply:
Followup?

Reply:
still needs a trailing newline though